### DISCUSSION TITLE
Unwatch suggestions not useful

### DISCUSSION THREAD

**Discussion by laktak** [community/community#11740] [Other Feature Feedback, Questions, & Ideas]
_Unwatch suggestions_ is still in beta so I'm not sure if this is a bug or a feature.

Every other week I get a big banner at the top of my notifications that says something like 
> "Overwhelmed with notifications? We have found [some stuff to turn off]." 
(sorry, forgot to take a screenshot).

I think it's nice to show this once to announce a new feature but I get this way too often that I now find it very annoying and would like to turn it off. To be clear - there is a X to dismiss the message but it will be back.

Also: notifications are not important enough to get their own feedback category? ;)

**Comment by alexjpwalker** [comment]
In my experience with Unwatch Suggestions, the repos flagged up by the feature typically do not correspond to repos that are actually triggering notifications - at least, no substantial amount of them.

More often, they flag up repos that have triggered maybe 1 or 2 notifications in the entirety of the past year. These are clearly not the repos that would be responsible for any feeling of being "overwhelmed by notifications".
